### DISCUSSION TITLE
Can you use the time value for the hidden field in future submits?

### DISCUSSION THREAD

**Discussion by Casmo** [spatie/laravel-honeypot#114] [Q&A]
When I copy an (encrypted) value from the `your_submission_date` and use that value the next day, my form is still submitted. It seems I can always use it in future submits because the code only checks if the date is not in the future. See [here](https://github.com/spatie/laravel-honeypot/blob/main/src/SpamProtection.php#L56).

Would it not been an idea to allow only X minutes ago as valid timestamp?

**Comment by parallels999** [comment]
Hi, feel free to do a PR to fix it, thanks
